llvm/test/CodeGen/AIE/aie2/schedule/postpipeliner/conv2d_bf16-1.mir

```diff
@@ -0,0 +1,238 @@
+# NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+
+# RUN: llc --mtriple=aie2 -O2 --start-before=postmisched %s \
+# RUN:   --debug-only=postpipeliner-summary -o - | FileCheck %s
```

Comment:
What is the purpose of `--debug-only=postpipeliner-summary` in this test?

Reply:
developer convenience. update_llc_tests shows it on stderr, and the last line reports the final II.